test/e2e/e2e-test/docker/browser/with-skywalking-client-js/Dockerfile.test-ui

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM node:10.23 AS builder
+
+## download and build skywalking client js
+ARG CLIENT_JS_COMMIT_HASH=04dd6b3b024f838b3b92002a370ce3b7f70638b4
```

Comment:
Do you need to consider merging this first? And use the latest commit id?
https://github.com/apache/skywalking-client-js/pull/42

Reply:
done